site/content/docs/contributing/project-structure.md

```diff
@@ -90,18 +90,18 @@ These programs are used to generate Kubernetes-like APIs. These programs are run
 To understand this process better, we need to keep in mind that kind’s configuration schema dictates how to bootstrap a Kubernetes cluster. The schema is defined in
 [kind/pkg/cluster/config](https://sigs.k8s.io/kind/pkg/cluster/config).
 In this directory, currently, you will see two subdirectories:
```

Comment:
nit: one subdirectory now?

Reply:
:man_shrugging: it will be two again pretty soon, this doc goes into a bit too much detail about the layout for an alpha codebase, it's pretty hard to keep it up to date.